compiler/backend/src/org/jetbrains/kotlin/codegen/inline/InlineCodegen.kt

```diff
@@ -234,7 +238,7 @@ abstract class InlineCodegen<out T : BaseExpressionCodegen>(
         val inliner = MethodInliner(
             node, parameters, info, FieldRemapper(null, null, parameters), isSameModule,
             "Method inlining " + sourceCompiler.callElementText,
-            SourceMapCopier(sourceMapper, nodeAndSmap.classSMAP, callSite.takeIf { !isCallOfFunctionInCorrespondingDefaultDispatch }),
+            SourceMapCopier(sourceMapper, nodeAndSmap.classSMAP, callSite),
```

Comment:
That parameter was not used for anything else; in the override, you can check that using IR structures

Reply:
Right. And with the custom call generator the parameter is not needed at all and I've gotten rid of it. :)